resources/views/emails/contactUs.blade.php

```diff
@@ -1,19 +1,25 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
- </head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+</head>
+
 <body>
   <p>User requested attention:</p>
   <p>User firstName: {{ $firstName }}</p>
   <p>User lastName: {{ $lastName }}</p>
   <p>User email: {{ $email }}</p>
   <p>User tel: {{ $tel }}</p>
-  <p>User sex: {{ $sex }}</p>
-  <p>User membership: {{ $membership }}</p>
-  <p>User sport: {{ $sport }}</p>
-  <p>User coach: {{ $coach }}</p>
+  <p>Sex interested in: {{ implode(',', $sex ?? []) }}</p>
```

Comment:
It looks pretty strange. Please explain your intention.

Reply:
Changed.